tests/integration/privateweave/kubernetes.tf

```diff
@@ -13,7 +13,7 @@ resource "aws_autoscaling_group" "bastion-privateweave-example-com" {
   launch_configuration = "${aws_launch_configuration.bastion-privateweave-example-com.id}"
   max_size = 1
   min_size = 1
-  vpc_zone_identifier = ["${aws_subnet.private-us-test-1a-privateweave-example-com.id}"]
+  vpc_zone_identifier = ["${aws_subnet.utility-us-test-1a-privateweave-example-com.id}"]
```

Comment:
We are not, but "normal" public & private subnets (where nodes live) are unqualified i.e. `us-east-1a.k8s.example.com` in both topologies.  We have to do this because when we're doing the version conversion we don't have access to the full cluster state.  It does have some advantages though - I think it might be possible to move from topology public <-> private, though I haven't tried it..